scout/server/blueprints/diagnoses/views.py

```diff
@@ -35,7 +35,8 @@ def count_diagnoses():
 @omim_bp.route("/api/v1/diagnoses")
 @public_endpoint
 def api_diagnoses():
-    """Return JSON data about OMIM diseases in the database."""
-
-    data = controllers.disease_terms(store)
+    """Return JSON data about diseases in the database."""
+    query = request.args.get("query") or None
```

Comment:
Add type hints for all the following variables

Reply:
Type hints have been added